frontend/src/App.js

```diff
@@ -33,73 +46,75 @@ const LoginContainer = () => (
   </div>
 )
 
-const ClientContainer = () => (
 
-  <div className="App-header" style={{ backgroundColor: '#CCCCCC' }}>
+const ClientContainer = () => (
+  <div className="App-header clientadvisorBackground">
```

Comment:
is this a style that's shared btwn client and advisor and thats why its in styles.css instead of a const in this file?